README.md

```diff
@@ -840,6 +841,95 @@ public class Handler : IRequiresAsyncInitialization
 public class Container : IContainer<Server> {}
 ```
 
+### `Owned<T>` support
+
+Using `Owned<T>` instead of `T` gives a component the control over when the `T` instance and all its dependencies are released back to the container while maintaining the component’s agnosticism about all the relative lifespans. Injecting `Owned<T>` or delegates returning `Owned<T>` is enabled out of the box, and so are the `AsyncOwned<T>` versions of these. `AsyncOwned<T>` will be required to be used instead of `Owned<T>` when the `T` in question or any of its dependencies require asynchronous disposal.
```

Comment:
> while maintaining the component’s agnosticism about all the relative lifespans

I'm not sure what this means?

Reply:
Removed the phrase to reduce clutter. Do you think it is clear enough?

Reply:
I can still see the phrase in the diff. Did you not push it or is there a caching issue on my end?

Reply:
Oops, missed `git add .` for my last edit. 😄

Reply:
I removed all the talk about lifespans and agnosticism with a paragraph about leaks and pooling. Do you think this is simple enough and gives the right hints for people looking for them?

Reply:
(I could remove the pooling sentence still, since technically pooling works with `Func<T>` if you don't care about leaks)